elasticsearch/templates/secret-cert.yaml

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
The certificate should be created and mounted only if a specific value is set to true (`{{- if .Values.createCert }}` for example) so people can keep providing their own certificates